app/views/content/train-to-be-a-teacher/teaching-internships.md

```diff
@@ -12,11 +12,11 @@ promo_content:
   - content/train-to-be-a-teacher/promos/eta-promo-internships
 backlink: /
 fullwidth: true
-navigation: 5.55
+navigation: 20:25
```

Comment:
I think you might need to be a bit careful here that you don't have conflicting numbers over the two PRs (quals and GSE)?